project/src/test/java/com/google/sps/DocumentTest.java

```diff
@@ -30,7 +30,8 @@ public void testGetters() {
     userIDs.add(100000L);
     ArrayList<Long> editorIDs = new ArrayList<Long>();
     ArrayList<Long> viewerIDs = new ArrayList<Long>();
-    Document doc = new Document("new_doc", "python", "x6723hbS", editorIDs, viewerIDs, 100000L, Folder.DEFAULT_FOLDER_ID);
+    ArrayList<Long> commentIDs = new ArrayList<Long>();
```

Comment:
please add more extensive testing for comment creation (eg if I create multiple comments, does the backend save them correctly)

Reply:
sounds good